src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -469,60 +561,6 @@ private static dynamic ReAssignComplexDerivedType(dynamic originalValue, Type ne
             return newOriginalNestedResource;
         }
 
-        /// <summary>
-        /// Copies the unchanged property values from the underlying entity (accessible via <see cref="GetInstance()" />)
-        /// to the <paramref name="original"/> entity.
-        /// </summary>
-        /// <param name="original">The entity to be updated.</param>
-        public void CopyUnchangedValues(TStructuralType original)
-        {
-            if (original == null)
-            {
-                throw Error.ArgumentNull(nameof(original));
-            }
-
-            if (!_structuredType.IsInstanceOfType(original))
-            {
-                throw Error.Argument(nameof(original), SRResources.DeltaTypeMismatch, _structuredType, original.GetType());
-            }
-
-            IEnumerable<PropertyAccessor<TStructuralType>> propertiesToCopy = GetUnchangedPropertyNames().Select(s => _allProperties[s]);
-            foreach (PropertyAccessor<TStructuralType> propertyToCopy in propertiesToCopy)
-            {
-                propertyToCopy.Copy(_instance, original);
-            }
-
-            CopyUnchangedDynamicValues(original);
-        }
-
-
-        /// <summary>
-        /// Overwrites the <paramref name="original"/> entity with the changes tracked by this Delta.
-        /// <remarks>The semantics of this operation are equivalent to a HTTP PATCH operation, hence the name.</remarks>
-        /// </summary>
-        /// <param name="original">The entity to be updated.</param>
-        /// <returns>The original value after Patching</returns>
-        public TStructuralType Patch(TStructuralType original)
-        {
-            if (IsComplexType)
-            {
-                original = ReAssignComplexDerivedType(original, _structuredType, original.GetType(), ExpectedClrType) as TStructuralType;                
-            }
-
-            return CopyChangedValues(original);           
-        }
-
-        /// <summary>
-        /// Overwrites the <paramref name="original"/> entity with the values stored in this Delta.
-        /// <remarks>The semantics of this operation are equivalent to a HTTP PUT operation, hence the name.</remarks>
-        /// </summary>
-        /// <param name="original">The entity to be updated.</param>
-        public void Put(TStructuralType original)
-        {
-            CopyChangedValues(original);
-            CopyUnchangedValues(original);
-        }
```

Comment:
remove the public APIs is breaking change

Reply:
fixed